xlml/apis/task.py

```diff
@@ -70,14 +70,39 @@ def run(self) -> DAGNode:
         group_id=self.task_test_config.benchmark_id, prefix_group_id=True
     ) as group:
       provision, queued_resource, ssh_keys = self.provision()
-      run_model = self.run_model(queued_resource, ssh_keys)
       post_process = self.post_process()
       clean_up = self.clean_up(queued_resource)
-
+      run_model = self.run_model(queued_resource, ssh_keys)
```

Comment:
Nit: could you move this change back to the line #73? which is before post_process.

Reply:
Done.